docs/src/main/asciidoc/_elytron/migrate/Properties_File_Based_Authentication_Migration.adoc

```diff
@@ -243,6 +243,104 @@ Which results in: -
 At this stage the authentication is the equivalent of the original
 configuration however now Elytron components are used exclusively.
 
+[[FileSystemRealm_Based_Authentication_Migration]]
+== Migrating to FileSystemRealm Based Authentication
+
+An alternative to using a properties-realm in Elytron is to use
```

Comment:
I think this paragraph should explicitly mention that this command can now be used to convert legacy properties files into an Elytron filesystem-realm.

Reply:
Modified the paragraph